infra/modules/feature-flags/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "service_name" {
+  type        = string
+  description = "The name of the service that the feature flagging system will be associated with"
+}
+
+variable "feature_flags" {
```

Comment:
Same as above, do we want to call out this is for boolean flags only?

Reply:
See comment above. By definition flags/toggles are booleans. I don't think see any value in supporting other types. And most people familiar with feature flags will think of them as booleans anyways.